data/tooltalk/AccountTools-Alarm-Calendar-AddAlarm-0.json

```diff
@@ -119,5 +117,17 @@
             "role": "user",
             "text": "Thanks, that's all for now."
         }
+    ],
+    "intended_apis_used": [
```

Comment:
Are `intended_apis_used` and `intended_suites_used` used anywhere? Or are they just to store old values of `suites_used` and `apis_used`? If it's the latter, then I think it's pretty safe to remove them.